plugins/hive-plugin/src/main/java/edu/berkeley/ground/plugins/hive/GroundStoreBase.java

```diff
@@ -0,0 +1,640 @@
+package edu.berkeley.ground.plugins.hive;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.hadoop.conf.Configurable;
+import org.apache.hadoop.hive.metastore.api.*; //TODO(krishna) fix
+import org.apache.hadoop.hive.metastore.FileMetadataHandler;
+import org.apache.hadoop.hive.metastore.RawStore;
+import org.apache.hadoop.hive.metastore.partition.spec.PartitionSpecProxy;
+import org.apache.thrift.TException;
+
+
+public abstract class GroundStoreBase implements RawStore, Configurable {
```

Comment:
This is to make the code more readable. GroundStore implements the functionality that we intend to support. The rest are stub implementations in GroundStoreBase.